frontend/src/app/features/assistance-selection/assistance-selection.component.html

```diff
@@ -0,0 +1,10 @@
+<h3>Select Your Desired Assistance Level on the Test:</h3>
+<div>
+  <ng-container *ngFor="let level of levels">
+    <button mat-raised-button color="primary"
+     (click)="handleSelectLevel(level)"
+     [id]="'id-' + level.key"
+     [ngClass]="{selected: level.key === selectedLevel}"
```

Comment:
The `[ngClass]` as written works, but for a single class, the preferred would be ...

```html
[class.selected]="level.key === selected"
```

Here and in the JavaScript, I'd use `selected` as we did with the other component to keep the language and naming similar where we can.